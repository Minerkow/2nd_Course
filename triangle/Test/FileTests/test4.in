20
 59.1376 85.3594 4.81706  82.2844 66.423 5.76945  73.2686 49.3166 0.817178 
 45.0823 87.9998 8.876  62.4836 81.2485 9.71432  59.2844 89.5384 3.04668 
 5.23725 21.3174 11.7664  6.94379 19.1104 15.8787  3.2435 23.8427 18.2609 
 52.1719 13.4103 12.1394  35.3289 8.4165 15.5382  55.5224 26.037 16.2897 
 50.8141 44.9552 12.5594  52.8519 44.7186 7.59113  51.5237 41.8799 11.7193 
 87.9747 31.0005 5.98248  93.8511 47.1231 83.1811  84.5574 80.1475 70.3464 
 -1.21299 33.5477 33.896  9.88118 24.3076 17.084  6.82301 49.0867 20.3444 
 40.2908 63.709 25.7291  11.0845 39.0411 59.0743  38.241 50.0441 63.0652 
 70.06 111.316 62.5433  95.5941 97.6392 53.5581  52.3186 106.136 84.4587 
 17.514 21.023 47.3311  18.1834 21.3229 47.2199  17.8586 21.7088 47.5566 
 25.0261 9.99708 83.3773  23.9136 9.62142 79.5894  23.4494 11.3403 80.9086 
 38.8989 28.7939 111.796  59.2421 37.2092 74.9616  51.3406 38.7945 110.477 
 -36.2884 44.8621 64.1062  -36.3883 28.4882 25.358  18.364 -6.45055 33.0412 
 11.9334 -2.01702 73.027  27.8249 3.6628 20.8101  6.76041 -4.90558 68.1023 
 28.521 79.2495 87.6522  6.63396 57.6585 48.2902  53.6229 71.3551 42.7333 
 43.7896 14.4747 88.6723  35.1146 60.5026 123.832  93.939 52.7484 107.96 
 26.959 53.4266 78.8454  24.2642 59.7883 -4.86403  35.46 15.8946 14.9535 
 28.7328 59.63 32.9969  24.9742 122.545 86.2421  -1.84767 54.9603 43.4851 
 39.3761 66.7734 20.1106  38.8937 64.3818 15.7274  40.7233 65.1889 14.3746 
 49.0618 99.5589 93.3998  64.8664 95.5961 107.725  67.6033 90.7744 106.821 