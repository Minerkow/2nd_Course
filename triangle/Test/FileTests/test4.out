7 
11 
15 
16